File: src/test/java/cn/enaium/cf4m/test/module/ModuleExtend.java
package cn.enaium.cf4m.test.module;

import cn.enaium.cf4m.annotation.module.Extend;

/**
 * @author Enaium
 */
@Extend
public class ModuleExtend {
    public String tag = null;
    public int age = 0;
    public Action func = null;

    public interface Action {
        void on();
    }
}

File: src/test/java/cn/enaium/cf4m/test/module/ModuleExtend.kt
package cn.enaium.cf4m.test.module

import cn.enaium.cf4m.annotation.module.Extend

/**
 * @author Enaium
 */
@Extend
class ModuleExtend {
    var tag: String? = null
    var age = 0
    var func: Action? = null

    interface Action {
        fun on()
    }
}